src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -100,8 +100,6 @@
         DEFAULT_SERVICE_PORT = {@context.getServiceConfig.getServicePort()}
         """The default port of the service."""
 
-        _CODE_GEN_NAME_VERSION = 'gapic/0.1.0'
-
         _GAX_VERSION = pkg_resources.get_distribution('google-gax').version
```

Comment:
It looks like `_GAX_VERSION` is now unused (and presumably can be derived in GAX itself); please remove.